docs/How-To-Open-a-Homebrew-Pull-Request.md

```diff
@@ -16,6 +16,15 @@ The type of change you want to make influences which of Homebrew's main reposito
 
 ### Core `brew` code pull request
 
+Before getting started, please note that we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future. If you don't meet notability requirements you might consider [creating your own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).
+
+Notability [auditor](https://github.com/Homebrew/brew/blob/638a3dc3a33c4c2029db058cf3d399008621e973/Library/Homebrew/utils/shared_audits.rb) will fail on repositories that do not meet:
+
+- **>29** days old
+- **>29** forks
+- **>29** watchers (Github only)
+- **>74** stars
+
```

Comment:
Hey @jrschumacher, thanks for the PR!

A link to the "Acceptable Formulae" page might be more suitable here. Also, this should be under "Formulae-related pull request". If you're feeling really nice: a link to "Acceptable Casks" under "Cask-related pull request" would be great, too!

Reply:
Happy to help. I'll try and get it done today.